css/portal-dashboard/activity-navigator.less

```diff
@@ -0,0 +1,68 @@
+@import "./variables";
+
+.activityNav {
+  background-color: @cc-teal-light6;
+  width: 100%;
+  flex: 0 0 50px;
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: 50px;
+  background-color: @cc-teal-light5;
+  user-select: none;
+  text-align: center;
+  border-bottom: 1px solid @cc-teal;
+
+  .navButtons {
+    display: flex;
+    align-items: center;
+    margin-left: 20px;
+
+    .button {
+      width: 32px;
+      height: 32px;
+      border-radius: 3px;
+      box-sizing: border-box;
+      border: solid 1.5px @cc-charcoal-light1;
+      background-color: @cc-teal-light5;
+      cursor: pointer;
+
+      &:nth-of-type(2) {
+        margin-left: 13px;
+        transform: rotate(180deg);
+      }
+
+      .icon {
+        width: 32px;
+        height: 32px;
+        margin-top: -2px;
+        margin-left: -2px;
+        fill: @cc-teal;
+      }
+
+      &:not(.disabled) {
+        &:hover {
+          background-color: @cc-teal-light3;
+        }
+        &:active {
+          background-color: @cc-teal;
+          border-color: white;
+
+          .icon {
+            fill: white;
+          }
+        }
+      }
+      &.disabled {
+        cursor: auto;
+        opacity: 0.35;
+      }
+    }
+  }
+
+  .title {
+    width: 78%;
```

Comment:
This width seems a little odd.  Doesn't the container change widths?  Is the title really always 78%?  But it seems fine in practice.  Let's just remember to keep an eye on it.